xsensmt/XsensMT.cpp

```diff
@@ -108,13 +108,29 @@ bool XsensMT::open(yarp::os::Searchable &config)
         yWarning() << "xsensmt -  Parameter \"frame_name\" not set. Using the same value as \"sensor_name\" for this parameter.";
     }
 
+    yDebug() << "xsensmt - test - xsensmt_period, check xsensmt_period: " << config.check("xsensmt_period");
+    yDebug() << "xsensmt - test - xsensmt_period, xsensmt_period is string: " << config.find("xsensmt_period").isString();
+    yDebug() << "xsensmt - test - xsensmt_period, xsensmt_period is int: " << config.find("xsensmt_period").isInt32();
+    yDebug() << "xsensmt - test - xsensmt_period, xsensmt_period is float: " << config.find("xsensmt_period").isFloat64();
+
+    if (config.check("xsensmt_period") && ( config.find("xsensmt_period").isInt32() || config.find("xsensmt_period").isFloat64()))
```

Comment:
Once you are ready, remember to add the parameter in https://github.com/robotology/yarp-device-xsensmt/blob/b31cba62d2b3f31a3f60d8353e658005b77fe7d4/xsensmt/XsensMT.h#L41 . Probably it could make sense to move that documentation in the README.md to increase its visibility, but that is not related to this PR.

Reply:
Ok, add the parameter to the table in https://github.com/robotology/yarp-device-xsensmt/pull/34/commits/28bdd1b1c310f84e74e5d67c6032cc3c6f8d07a6